nekoyume/block.py

```diff
@@ -204,23 +206,28 @@ def find_branch_point(value: int, high: int):
             mid = int((value + high) / 2)
             response = get((f"{node.url}{Node.get_blocks_endpoint}/"
                             f"{mid}"))
+            if response.status_code != 200:
+                return
```

Comment:
I think we should raise error instead of change return type (`int` to `Optional[int]`).